.github/workflows/check-links-cron.yaml

```
@@ -0,0 +1,56 @@
+name: Check Documentation Links
+
+on:
+  schedule:
+    - cron: '0 9 * * 1'
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/check-links-cron.yaml'
+
+permissions: read-all
+
+jobs:
+  links-checker:
+    name: Documentation links checker
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+        with:
+          persist-credentials: false 
+      - name: Check Documentation links
+        uses: docker://quay.io/cilium/docs-builder:4b194c0facc5d1d3b9c0dfef3529ee7c661a6b73@sha256:095cf9621cf46c6c878e25c3a98f02ef3d0aed1154936663e352fd3fe5e7613d
+        with:
+          entrypoint: ./Documentation/check-build.sh

```

```suggestion
          entrypoint: ./Documentation/check-links.sh

```

My bad, wrong suggestion.